src/CommunityPatchLauncher/Commands/Os/StartSettlersConfigCommand.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using CommunityPatchLauncherFramework.Settings.Manager;
+
+namespace CommunityPatchLauncher.Commands.Os
+{
+    class StartSettlersConfigCommand : StartProgramCommand
```

Comment:
The `StartProgrammCommand` can be preconfigured via constructor. Is this a option in this situation?

Reply:
If this is not an option, the class is missing a summary block